components/pedigree/resources/src/main/resources/pedigree/view/personVisuals.js

```diff
@@ -811,12 +811,17 @@ define([
                     this.getAgeLabel().show();
                     labels.push(this.getAgeLabel());
                 }
+                this.getExternalIDLabel() && labels.push(this.getExternalIDLabel());
```

Comment:
Hm.. In JIRA issue external ID isn't mentioned. Can it be considered as sensitive info?